packages/web3-eth-accounts/src/account.ts

```diff
@@ -26,12 +177,12 @@ export const privateKeyToAccount = (
 ): {
 	address: string;
 	privateKey: string;
-	signTransaction: () => boolean; // From 1.x
-	sign: () => boolean;
+	signTransaction: signTransactionFunction; // From 1.x
+	sign: signFunction;
 	encrypt: () => boolean;
 } => {
```

Comment:
You don't need to specify the return object like this, TypeScript will automatically infer it